waiter/src/waiter/utils.clj

```diff
@@ -472,3 +473,8 @@
   "Parses the request header to determine if debug mode has been enabled."
   [request]
   (boolean (get-in request [:headers "x-waiter-debug"])))
+
+(defn keyset
```

Comment:
Seems worse to me than just `(set (key m))` in all cases.  Should we also make `keyvec`, `valvec`, `valset`, `keylist`, `vallist`, etc?

Reply:
Removed `utils/keyset`